src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -1018,32 +1022,37 @@ export function submitAppointmentOrRequest(history) {
 
       try {
         let requestData;
-        if (isCommunityCare) {
+        if (featureVAOSServiceRequests && isCommunityCare) {
```

Comment:
This is the main switch point, where we send requests to the VAOS service instead of the current services